tests/test_ui_polish.py

```
@@ -0,0 +1,149 @@
+        '.list-view .file-item',
+        """items => {
+            const row = items.find(el => el.querySelector('.file-name-col').innerText.includes('空文件夹'));
+            if (!row) return null;
+            return [...row.querySelectorAll('.file-action-col a, .file-action-col button')]
+                     .map(a => (a.textContent || '').trim());
+        }""")
+    assert labels is not None, '应找到空文件夹行'
+    text = ' '.join(labels)
+    assert '打包下载' not in text, f'无图目录不应有打包下载: {labels}'
+    assert '看本' not in text, f'无图目录不应有看本: {labels}'
+    assert any('重命名' in l for l in labels) and any('移动' in l for l in labels), \
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)